pkgs/applications/editors/kakoune/default.nix

```diff
@@ -1,18 +1,18 @@
-{ lib, stdenv, fetchFromGitHub, ncurses, asciidoc, docbook_xsl, libxslt, pkg-config }:
+{ lib, stdenv, fetchFromGitHub, asciidoc, docbook_xsl, libxslt, pkg-config }:
 
 with lib;
 
 stdenv.mkDerivation rec {
   pname = "kakoune-unwrapped";
-  version = "2020.09.01";
+  version = "2021.08.28";
   src = fetchFromGitHub {
     repo = "kakoune";
     owner = "mawww";
     rev = "v${version}";
-    sha256 = "091qzk0qs7hql0q51hix99srgma35mhdnjfd5ncfba1bmc1h8x5i";
+    sha256 = "13kc68vkrzg89khir6ayyxgbnmz16dhippcnw09hhzxivf5ayzpy";
   };
   nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ ncurses asciidoc docbook_xsl libxslt ];
+  buildInputs = [ asciidoc docbook_xsl libxslt ];
```

Comment:
It also looks like none of those are needed anymore:

```suggestion
```

Reply:
Oh cool. I don't fully understand what these were there for in the first place (perhaps related to the wrapper that does something with the docs?) but at least it seems to build and execute without.

Reply:
They used to be used to build man pages.